src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -2126,11 +2133,11 @@
     <define name="poem.line">
         <element name="span">
             <attribute name="class">
-                <choice>
+                <oneOrMore>
```

Comment:
My thought here was to always require `line`, and make it optional to also have either `line_indent` or `line_longindent`. But this is probably fine as well, so that we're not too strict about it.